gravitee-apim-console-webui/src/management/api/api-score/api-score.component.html

```diff
@@ -0,0 +1,67 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+
+<mat-card class="mat-card">
+  <mat-card-header class="card-header">
+    <mat-card-title>
+      {{ api?.name }}
+      @if (!isLoading && apiScore.all) {
+        <span class="gio-badge-success"><mat-icon class="gio-left" svgIcon="gio:shield-check"></mat-icon>90%</span>
+        <span class="gio-badge-success"><mat-icon class="gio-left" svgIcon="gio:trending-up"></mat-icon>+0.7</span>
+      }
+    </mat-card-title>
+
+    @if (apiScore.lastEvaluation) {
+      <mat-card-subtitle> Last evaluated {{ apiScore.lastEvaluation }} minute ago</mat-card-subtitle>
+    }
+
+    <div class="card-header__actions">
+      <button mat-raised-button color="primary" data-testid="discover-button" (click)="evaluate()">Evaluate</button>
+    </div>
+  </mat-card-header>
+
+  <div class="separator-hr"></div>
+
+  <mat-card-content class="card-content">
+    @if (apiScore.all !== 0) {
+      <section>
+        <mat-button-toggle-group class="gio-button-toggle-group" name="status" aria-label="" [value]="status">
+          <mat-button-toggle value="all">All ({{ apiScore.all }})</mat-button-toggle>
+          <mat-button-toggle value="errors">Errors ({{ apiScore.errors }})</mat-button-toggle>
+          <mat-button-toggle value="warnings">Warnings ({{ apiScore.warnings }})</mat-button-toggle>
+          <mat-button-toggle value="infos">Infos ({{ apiScore.infos }})</mat-button-toggle>
+          <mat-button-toggle value="hints">Hints ({{ apiScore.hints }})</mat-button-toggle>
+        </mat-button-toggle-group>
+      </section>
+    }
+
+    <section class="api-scores-lists">
+      @if (apiScore.all === 0) {
+        <gio-card-empty-state
+          icon="rocket"
+          title="All clear"
+          [subtitle]="'There is no recommendations or issues for your API. \n Everything looks great!'"
+        ></gio-card-empty-state>
+      }
+
+      @for (apiScoreItem of apiScore.scoreLists; track apiScoreItem.name) {
+        <app-api-score-list [scoreList]="apiScoreItem"></app-api-score-list>
+      }
```

Comment:
Maybe that can be writen with

```
@for(...) {
    <app-api-score-list>...
} @empty {
    <gio-card-empty-state>...
}
```

Reply:
good comment, however we still show api score lists (they are empty lists wiht headers) when there are no score commnets . So it is never actually empy.